test/tagger/index.test.ts

```diff
@@ -44,7 +37,7 @@ describe("Tagger", () => {
   });
 
   afterEach(() => {
-    window.XMLHttpRequest = oldXMLHttpRequest;
+    xhrSpy.mockRestore();
```

Comment:
There's a global mock restore, so this isn't necessary. https://github.com/crookedneighbor/shambleshark/pull/45/commits/20b335e1337bacee1c08534ca70ce86574942634